main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "3.62.1"
+      version = "3.86.0"
```

Comment:
Is there a specific reason, why the version of the azurerm provider is updated or just do be up to date?